CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
  - Adapting SpriteBatchComponent constructors to match the pattern used on other components
  - Renamed `Composition` to `ImageComposition` to prevent confusion with the composition component
  - Added `rotation` and `anchor` arguments to `ImageComposition.add`
+ - Added `Image` extensions
+ - Added `Color` extensions
```

Comment:
I think an example could be useful? but it can be a followup on the new repo instead of adding examples on the old structure